examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -138,21 +137,11 @@ def loss_fxn(ff_params, batch: List[Tuple[RelativeFreeEnergy, RBFEModel]]):
         pred_ddG, stage_results = model.predict(ff_params, rfe.mol_a, rfe.mol_b, rfe.core)
         all_results.extend(list(stage_results))
         preds.append(pred_ddG)
-    # If the edges in the batch are within a cycle, correct the ddGs
-    indices = jnp.asarray(indices)
-    ind_l, ind_r = indices.T
-    try:
-        layer = construct_mle_layer(len(index), indices)
-        corrected_dg = layer(jnp.asarray(preds))
-        cycle_corrected_rbfes = corrected_dg[ind_r] - corrected_dg[ind_l]
-    except DisconnectedEdgesError:
-        # Provided non connected graph, use the ddGs directly
-        cycle_corrected_rbfes = jnp.asarray(preds)
     labels = jnp.asarray([rfe.label for rfe, _ in batch])
-    loss = pseudo_huber_loss(cycle_corrected_rbfes - labels)
+    loss = pseudo_huber_loss(preds - labels)
```

Comment:
Does this need `psuedo_huber_loss(jnp.asarray(preds) - labels)`?

Reply:
Yes -- thanks for catching. Fixed in https://github.com/proteneer/timemachine/pull/523/commits/67b5c988c9dba4f4902eeb7aace3024ecc6bb4ef